webapps/camunda-commons-ui/lib/widgets/selection-type/cam-widget-selection-type.template.html

```diff
@@ -0,0 +1,50 @@
+<div class="form-inline">
+  <div>
+    <label for="instance"></label>
+    <input type="radio"
+           id="instance"
+           name="selection_type"
+           value="INSTANCE"
+           ng-model="selectionType"
+           ng-change="updateSelectionType(selectionType)" />
+    {{ 'INSTANCE' | translate }}
+    <span class="glyphicon glyphicon-question-sign"
+          uib-tooltip="{{ 'CAM_WIDGET_SELECTION_TYPE_INSTANCE_TOOLTIP' | translate }}"
+          tooltip-placement="right"></span>
+  </div>
+  <div>
+    <label for="query"></label>
+    <input type="radio"
+           id="query"
+           name="selection_type"
+           value="QUERY"
+           ng-model="selectionType"
+           ng-change="updateSelectionType(selectionType)" />
+    {{ 'QUERY' | translate }}
+    <span class="glyphicon glyphicon-question-sign"
+          uib-tooltip="{{ 'CAM_WIDGET_SELECTION_TYPE_QUERY_TOOLTIP' | translate }}"
+          tooltip-placement="right"></span>
+  </div>   
+</div>
+
+<div class="instance"
+     ng-if="selectionType === 'INSTANCE' && toggleState === 'on' && totalInstancesCount > 12">
+  <div class="alert alert-info">
+    <p>{{ 'CAM_WIDGET_SELECTION_TYPE_INSTANCE_NOTICE_PLACEHOLDER' | translate }} {{ selectedInstancesCount }} {{ 'INSTANCES' | translate }}. </p><br />
+    <p>
+      <a ng-click="updateSelectionType('QUERY')">{{ 'CAM_WIDGET_SELECTION_TYPE_INSTANCE_NOTICE_LINK' | translate }} {{ totalInstancesCount }} {{ 'INSTANCES' | translate }}.</a>
+      <strong>{{ 'WARNING' | translate }}:</strong> {{ 'CAM_WIDGET_SELECTION_TYPE_NOTICE_WARNING_PLACEHOLDER' | translate }}
```

Comment:
```suggestion
```
I wouldn't display the warning before all instances have been selected. I think it is sufficient to only show it in the case of selection type "query" .